ci/test_full.sh

```diff
@@ -30,6 +30,7 @@ fi
 FEATURES=(libm serde)
 check_version 1.34 && FEATURES+=(bytemuck)
 check_version 1.36 && FEATURES+=(rand)
+check_version 1.54 && FEATURES+=(rkyv)
```

Comment:
Should add `bytecheck` here too. (assuming it has the same MSRV? or close enough?)

```suggestion
check_version 1.54 && FEATURES+=(rkyv bytecheck)
```